<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addRequest</class>
 <widget class="QDialog" name="addRequest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#addRequest{
	background-image:url(img/addRequest.jpg);
}
QTextEdit{
	background-color:rgb(235,199,255);
}  
     
QLineEdit{
	background-color:rgb(235,199,255);
}  
</string>
  </property>
  <widget class="QComboBox" name="proName">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>78</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Describe">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>180</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>116</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>需求类型：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnConfirm">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="dailyInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;黑体&quot;;
color:rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>需求名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>146</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>需求当前状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>需求描述：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>项目名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="requestName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="requestType">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>114</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>功能性需求</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>非功能性需求</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="requestState">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>145</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>未完成</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>已完成</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Info">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;黑体&quot;;
color:rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
